db/migrate/20191111234342_add_first_name_last_name_email_password_date_of_birth_and_role_to_user.rb

```diff
@@ -2,8 +2,6 @@ class AddFirstNameLastNameEmailPasswordDateOfBirthAndRoleToUser < ActiveRecord::
   def change
     add_column :users, :first_name, :string
     add_column :users, :last_name, :string
-    add_column :users, :email, :string
-    add_column :users, :password, :string
     add_column :users, :date_of_birth, :string
```

Comment:
is the date_of_birth a string?

Reply:
nice catch -- closed pull request will recommit with column change